native/sapp-windows/src/lib.rs

```diff
@@ -464,7 +468,37 @@ pub unsafe fn sapp_mouse_shown() -> bool {
     cursor_info.flags & CURSOR_SHOWING != 0
 }
 
-pub unsafe fn sapp_set_cursor_grab(mut _grab: bool) {}
+pub unsafe fn sapp_set_cursor_grab(grab: bool) {
+    if grab == _sapp.cursor_grabbed {
+        return;
+    }
+
+    _sapp.cursor_grabbed = grab;
+
+    let mut rid: RAWINPUTDEVICE = RAWINPUTDEVICE {
+        usUsagePage: 0x01,
```

Comment:
Maybe use constants from winapi here?
 
 winapi::shared::hidusage::HID_USAGE_GENERIC_POINTER and HID_USAGE_GENERIC_MOUSE
 ```
HID_USAGE_GENERIC_POINTER: USAGE = 0x01;
HID_USAGE_GENERIC_MOUSE: USAGE = 0x02;
```